modules/data/log/tmpl/index.tmpl

```diff
@@ -0,0 +1,40 @@
+<? I18N znc-log ?>
+<? INC Header.tmpl ?>
+
```

Comment:
As log module has some settings which are not accessible via this UI, please mention that here. Or you can implement them if you wish.

Reply:
Are you referring to the arguments I saw mentioned in the module pages where you enable the log module? I didn't even think of those since they already had a method of activation.

Reply:
No, not the arguments. Check `/msg *log help`

Reply:
Ok. I think I'll try to implement these.

Reply:
The rules and settings configuration should now be usable from all 3 module modes but the log related functionality is not available to the global module.